central/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM openjdk:8
 
+ENV GLOWROOT_OPTS="-XX:+PrintFlagsFinal -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap"
```

Comment:
`-XX:+PrintFlagsFinal` adds a lot of verbosity

Reply:
You're right, no need for such verbosity